docs/source/install/profile.rst

```diff
@@ -194,6 +194,23 @@ Additionally, you need to make the ``rsync`` command available as written below.
 .. literalinclude:: profiles/lawrencium-lbnl/picongpu.profile.example
    :language: bash
 
+Cori (NERSC)
+------------
+
+**System overview:** `link <https://www.nersc.gov/users/computational-systems/cori/configuration/>`_
+
+**User guide:** `link <https://docs.nersc.gov/>`_
+
+**Production directory:** ``$SCRATCH`` (`link <https://www.nersc.gov/users/storage-and-file-systems/>`_).
+
+For these profiles to work, you need to download the :ref:`PIConGPU source code <install-dependencies-picongpu>` and install :ref:`PNGwriter and libSplash <install-dependencies>` manually.
+
+Queue: regular (Intel  Intel Xeon Phi - Knights Landing)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Guess that was my misunderstanding: should the `.tpl` and `_profile.example` file names be the same as the queue name? Here it is `regular` vs. `knl` (although `knl` makes more sense definitely)

Reply:
Yes, in the past these were identical. Somehow we have clusters these days (Taurus, Cori) that use the same queue for various hardware and specify the hardware via constrains (e.g. Slurm `-C`).

I think using the device hardware for the `.tpl` make sense, but we might want to change the sub-titles in this profile at some point. But it's still clear and fully described here, so I think the current way works.